README.md

```diff
@@ -222,105 +222,105 @@ environment:
 
 ### Timezone
 
-The default timezone is not defined (aka `UTC`). You can modify the default timezone by the `TZ` environment variable
-with timezone name as value (e.g.: `Europe/Prague`).
+The default timezone is not defined (`UTC` will be used). You can modify the default timezone by setting the `TZ` environment variable
+to the desired timezone name (e.g.: `Europe/Prague`).
 
-You can put it directly with `docker run`:
+It can also be specified directly with `docker run`:
 
 ```shell
 docker run -it --rm -e TZ=Europe/Prague jakubboucek/lamp-devstack-php
 ```
 
-You can also put it to `docker-compose.yml` file:
+Or in your `docker-compose.yml` file:
 
 ```yaml
 environment:
     TZ: Europe/Prague
 ```
 
-The `TZ` environment variable is recognized by Linux, by that variable you modify the default timezone for the whole Linux
+The `TZ` environment variable is recognized by Linux tools as well, by creating the variable you modify the default timezone for the whole Linux
 operating system, PHP, and also MySQL.
 
-### Temporary, Upload and Session storage directory
+### temporary, upload and session storage directory
 
-PHP is using native Linux temporary directory for all own temporary files (inlcuding Session and Upload storage). This
-image does not provide custom way to modify them. You can use the `TEMPDIR` environment variable to modify all of them.
+PHP is using native Linux temporary directory for all own temporary files (inlcuding session and upload storage). This
+image does not provide any custom way to modify them. You can use the `TEMPDIR` environment variable to modify all of them.
 
-You can put it directly with `docker run`:
+You can specify it directly with `docker run`:
 
 ```shell
 docker run -it --rm -e TEMPDIR=/var/www/temp jakubboucek/lamp-devstack-php
 ```
 
-You can also put it to `docker-compose.yml` file:
+Or in the `docker-compose.yml` file:
 
 ```yaml
 environment:
   TEMPDIR: /var/www/temp
 ```
 
-Note: The directory MUST already exists and MUST be writable to all users (`0777`), otherwise PHP can be unstable or
-can lost data (e.g. Sessions data). Moving temporary directory to volume binded to the Host can have a big impact
+Note: The directory MUST already exists and MUST be writable for all users (`0777`), otherwise PHP can be unstable or
+can lose data (e.g. sessions data). Moving temporary directory to volume shared with the Host can have a big impact
```

Comment:
"the" temporary directory to "the" volume

Reply:
Spíš to "a" volume

Reply:
Může být, na to nemám dost hlubokou expertízu v týhle oblasti :) Tipoval jsem, že tam bude max 1 shared volume.